src/test/kotlin/no/nav/familie/ef/søknad/person/mapper/SøkerinfoMapperTest.kt

```diff
@@ -134,17 +135,62 @@ internal class SøkerinfoMapperTest {
         assertThat(tilDto.harAdressesperre == false)
     }
 
+    @Test
+    fun `skal kalkulere alder riktig for en person født i 2000 med D-nummer`() {
+        val fødselsdato = LocalDate.of(2000, 1, 1)
+        val fødselsnummer = FnrGenerator.generer(fødselsdato, erDnummer = true)
+        every { EksternBrukerUtils.hentFnrFraToken() } returns fødselsnummer
+
+        val pdlSøker = opprettPdlSøker()
+        val person = søkerinfoMapper.mapTilSøkerinfo(pdlSøker, emptyMap(), emptyMap()).søker
+        val forventetAlder = Period.between(fødselsdato, LocalDate.now()).years
+
+        assertThat(person.alder).isEqualTo(forventetAlder)
+    }
+
+    @Test
+    fun `skal kalkulere alder riktig for en person født i 1990 med D-nummer`() {
+        val fødselsdato = LocalDate.of(1990, 1, 1)
+        val fødselsnummer = FnrGenerator.generer(fødselsdato, erDnummer = true)
+        every { EksternBrukerUtils.hentFnrFraToken() } returns fødselsnummer
+
+        val pdlSøker = opprettPdlSøker()
+        val person = søkerinfoMapper.mapTilSøkerinfo(pdlSøker, emptyMap(), emptyMap()).søker
+        val forventetAlder = Period.between(fødselsdato, LocalDate.now()).years
+
+        assertThat(person.alder).isEqualTo(forventetAlder)
+    }
+
+    @Test
+    fun `skal kalkulere alder riktig for en person født i 1990`() {
+        val fødselsdato = LocalDate.of(1990, 1, 1)
+        val fødselsnummer = FnrGenerator.generer(fødselsdato)
+        every { EksternBrukerUtils.hentFnrFraToken() } returns fødselsnummer
+
+        val pdlSøker = opprettPdlSøker()
+        val person = søkerinfoMapper.mapTilSøkerinfo(pdlSøker, emptyMap(), emptyMap()).søker
+        val forventetAlder = Period.between(fødselsdato, LocalDate.now()).years
+
+        assertThat(person.alder).isEqualTo(forventetAlder)
+    }
```

Comment:
Denne testen sjekker nøyaktig det samme som testen under, så denne kan fjernes

Reply:
Egentlig ikke, siden det er fnr i stedet for dnr